taxi/urls.py

```diff
@@ -2,26 +2,72 @@
 
 from .views import (
     index,
-    CarListView,
-    CarDetailView,
     DriverListView,
     DriverDetailView,
     ManufacturerListView,
+    ManufacturerCreateView,
+    ManufacturerUpdateView,
+    ManufacturerDeleteView,
+    CarListView,
+    CarDetailView,
+    CarCreateView,
+    CarUpdateView,
+    CarDeleteView,
 )
 
 urlpatterns = [
-    path("", index, name="index"),
+
     path(
         "manufacturers/",
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/update",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>/delete",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete",
+    ),
+
     path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path(
+        "cars/create/",
+        CarCreateView.as_view(),
+        name="car-create",
+    ),
+
+    path(
+        "cars/<int:pk>/",
+        CarDetailView.as_view(),
+        name="car-detail",
+    ),
+
+    path(
+        "cars/<int:pk>/update",
```

Comment:
path must end at /
check in all places

Reply:
Thanks, I"ll fix all places where i missed /.